content/en/docs/tasks/configure-pod-container/resize-pod-resources.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Resize CPU and Memory Resources assigned to Containers
```

Comment:
I am not sure why it doesn't show up in left pane, maybe some index needs to be updated as well:

<img width="418" height="1305" alt="image" src="https://github.com/user-attachments/assets/a3f1019e-85ee-4cb9-9b5f-aa6de7816b82" />

https://deploy-preview-52928--kubernetes-io-main-staging.netlify.app/docs/tasks/configure-pod-container/resize-pod-resources/